src/components/Banner.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import styled from 'styled-components';
+import { graphql, StaticQuery } from 'gatsby';
+import { Card, Image } from 'semantic-ui-react';
+
+class Banner extends React.Component {
+
+  renderLogos = allLogos => (
+    <Card fluid >
+      <Card.Content className="card">
+        {this.getRandom(allLogos, 4).map(logo => {
+        if(logo.node.image.file.url != null){
+          return(
+            <Image className="image" key={logo.node.id} size="small" src={logo.node.image.file.url} />
+          );
+        }
+        return null;
+      })}
+      </Card.Content>
+    </Card>
+  )
+
+  getRandom = (arr, n) => {
+    var result = new Array(n),
+    len = arr.length,
+    taken = new Array(len);
+
+    if(n > len)
+      return arr;
+
+    while(n--){
+      var x = Math.floor(Math.random() * len);
+      result[n] = arr[x in taken ? taken[x] : x];
+      taken[x] = --len in taken ? taken[len] : len;
+    }
+    return result;
+  }
+
+  render(){
+    return(
+      <BannerWrapper>
+        {this.renderLogos(this.props.data.allContentfulBannerLogo.edges)}
+        </BannerWrapper>
+      );
+  }
+}
+
+const BannerWrapper = styled.div`
+  padding-bottom: 3em;
+  .card{
```

Comment:
Please make separate styled components for this as outlined [here](https://www.styled-components.com/docs/basics#coming-from-css)

Reply:
Tried that already, by putting it in a component, the padding won't appear.

Reply:
I tried it myself as well and couldn't get it to work. Too bad